src/SettingsView/EventType.js

```diff
@@ -29,13 +29,19 @@ class EventType extends ConnectionComponent {
 
 	render() {
 		const allFetches = PromiseState.all([this.props.queries, this.props.attributes]);
+		const optionalActions = this.props.deleteQueryeResponse;
+		console.log(optionalActions);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AS2228)